crates/zlog/src/sink.rs

```diff
@@ -45,6 +47,12 @@ pub fn init_output_stdout() {
     }
 }
 
+pub fn init_output_stderr() {
+    unsafe {
+        ENABLED_SINKS_STDOUT = true;
```

Comment:
Seems like we should be setting `ENABLED_SINKS_STDERR` here:

```suggestion
        ENABLED_SINKS_STDERR = true;
```

Reply:
This wasn't even AI. Thank you